core/src/main/kotlin/dev/hotwire/core/turbo/config/PathConfigurationLoader.kt

```diff
@@ -63,10 +63,12 @@ internal class PathConfigurationLoader(val context: Context) : CoroutineScope {
         repository.cacheConfigurationForUrl(context, url, pathConfiguration)
     }
 
-    private fun load(json: String) = try {
-        json.toObject(object : TypeToken<PathConfiguration>() {})
-    } catch(e: Exception) {
-        logError("pathConfiguredFailedToParse", e)
-        null
+    fun load(json: String): PathConfiguration? {
+     return try {
+            json.toObject(object : TypeToken<PathConfiguration>() {})
+        } catch (e: Exception) {
+            logError("pathConfiguredFailedToParse", e)
+            null
+        }
```

Comment:
This doesn't actually change any logic here. In Kotlin, `= try` is just shorthand for `return try`. Looking at this closer, we're already catching parsing exceptions correctly, just with a slightly different implementation from `turbo-android`.

Reply:
Yes, that's what I noticed when I started this. I meant to look into it further, but I became busy with work. 

Thanks for coming back to it.